toolchain/check/node_stack.h

```diff
@@ -385,8 +378,8 @@ class NodeStack {
 
   // Translate a parse node category to the enum ID kind it should always
   // provide, if it is consistent.
-  static constexpr auto ParseNodeCategoryToIdKind(Parse::NodeCategory category,
-                                                  bool for_node_kind)
+  static constexpr auto NodeIdCategoryToIdKind(Parse::NodeCategory category,
```

Comment:
Should this be `NodeCategoryToIdKind`?

Reply:
Yes, fixed. Also verified there are no other "NodeIdCategory" occurrences.